### ISSUE TITLE
authorization error in digest with quoted query parameters

### ISSUE THREAD

**Issue by tzejohn** [a85/POSTMan-Chrome-Extension#3281]
## App Details:
```
Postman for Chrome
Version 5.0.2
win / x86-32
Chrome 57.0.2987.133
```

## Issue Report:
 1. Did you encounter this recently, or has this bug always been there:
Just experienced it, but it was probably there from before.
 2. Expected behaviour:
I expect that using digest authentication for a GET request where a query parameter has quotes on it would work and not result in a (401) Unauthorized error. When the request gets to the end point the parameter is escaped. So a request in Postman of `http://10.239.12.203:8200/v1/search?q="calcium"` actually shows up in the access log of the server as `GET /v1/search?q=%22calcium%22 HTTP/1.1`. However, the value of the Authorization header is `Digest username="admin", realm="public", nonce="", uri="/v1/search?q="calcium"", response="e87751ba630e9ad8712aca3572277cd8", opaque=""`. I think this results in a different code that expected, and results in a 401. If I instead actually use the URL `http://10.239.12.203:8200/v1/search?q=%22calcium%22`, it works fine. Using the double quotes works fine in Restlet and Curl.

**Comment by sdnts** [comment]
Hi @tzejohn Can you try this on our [native apps](https://getpostman.com/apps) as well and check if you still get this error?

**Comment by tzejohn** [comment]
Yes. I tried this with the Mac native app, and it behaves the same way.

Postman for Mac
Version 5.0.2
OS X 15.6.0 / x64

**Comment by sdnts** [comment]
Thanks for confirming. We'll look into this and let you know here when a fix is out :)

**Comment by codenirvana** [comment]
Fixed in the latest Postman App (v7.1.0).
